frontend/src/app.css

```diff
@@ -43,3 +43,14 @@
     columns: 5rem auto;
   }
 }
+
+@variants {
```

Comment:
I'm not sure that this is a correct use of `@variants`. Please see https://tailwindcss.com/docs/functions-and-directives#variants. You are not listing any variant in the `@variants` directive. What is the use of this directive here?